blazor/datagrid/column-chooser.md

```diff
@@ -9,117 +9,436 @@ documentation: ug
+                {
+                    Orders.Add(new OrderData(10248,  "France", 33.33,new DateTime(1996,07,07), "Reims"));
+                    Orders.Add(new OrderData(10249,  "Germany", 89.76, new DateTime(1996, 07, 12), "Münster"));
+                    Orders.Add(new OrderData(10250,  "Brazil",78.67, new DateTime(1996, 07, 13), "Rio de Janeiro"));
+                    Orders.Add(new OrderData(10251,  "Belgium", 55.65, new DateTime(1996, 07, 14), "Lyon"));
+                    Orders.Add(new OrderData(10252, "Venezuela",11.09, new DateTime(1996, 07, 15), "Charleroi"));
+                    Orders.Add(new OrderData(10253, "Venezuela",98.98, new DateTime(1996, 07, 16), "Lyon"));
+                    Orders.Add(new OrderData(10254, "Belgium", 78.75, new DateTime(1996, 07, 17), "Rio de Janeiro"));
+                    Orders.Add(new OrderData(10255, "Germany", 44.07, new DateTime(1996, 07, 18), "Münster"));
+                    Orders.Add(new OrderData(10256, "France", 67.74, new DateTime(1996, 07, 19), "Reims"));
+                    code += 5;
+                }
+            }
+            return Orders;
+        }
+        public int? OrderID { get; set; }                
+        public DateTime OrderDate { get; set; }      
+        public string ShipCountry { get; set; }
+        public double Freight { get; set; }
+        public string ShipCity { get; set; }
+    }  
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hjLgCMZLiJiGNLIf?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+
+## Change default search operator of the column chooser
 
-    public class EmployeeData
+The column chooser dialog in the Syncfusion Blazor Grid provides a search box that allows you to search for column names. By default, the search functionality uses the **StartsWith** operator to match columns and display the results in the column chooser dialog. However, there might be cases where you need to change the default search operator to achieve more precise data matching.
+
+To change the default search operator of the column chooser in Syncfusion Grid, you need to use the [Operator](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Operator.html) property of the [GridColumnChooserSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumnChooserSettings.html) class.
+
+Here’s an example of how to change the default search operator of the column chooser to **Contains** in the Blazor Grid:
+
+{% tabs %}
+{% highlight razor tabtitle="Index.razor" %}
+@using Syncfusion.Blazor.Grids
+@using BlazorApp1.Data
+
+<SfGrid ID="Grid" DataSource="@Orders" ShowColumnChooser="true" Toolbar=@ToolbarItems>
+    <GridColumnChooserSettings Operator="Operator.Contains"></GridColumnChooserSettings>
+    <GridColumns>
+        <GridColumn Field=@nameof(OrderData.OrderID) HeaderText="Order ID" ShowInColumnChooser="false" TextAlign="TextAlign.Right" Width="120"></GridColumn>
```

Comment:
ShowInColumnChooser="false" not required here.

Reply:
removed